.github/workflows/check-generate-api.yml

```diff
@@ -0,0 +1,50 @@
+# SPDX-FileCopyrightText: (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+---
```

Comment:
Run `yamllint` on this, and generally the document start should be first line of the file.

Reply:
Removed newline before `---`
yamllint has only false positives and length errors related to CI action hash number
![image](https://github.com/user-attachments/assets/19901778-3e9f-4b4e-9fc4-d11aaf1c43ce)